niceman/distributions/tests/test_debian.py

```diff
@@ -54,6 +56,34 @@ def test_dpkg_manager_identify_packages():
         assert False, "A non-local origin must be found"
 
 
+@pytest.mark.slow
```

Comment:
I guess we would never know how long it takes on travis unless we run it there, right? ;)

Reply:
might also be a good time to try to profile our code since dpkg-query itself is quite fast and I don't remember if we otherwise analyzed our current implementation to make it any faster (might be doing the same thing over and over again).  The simplest initial step might be to use `@line_profile` helper for e.g. `identify_packages_from_files`

Reply:
BTW -- it did finish for me without crashing! test failed since we do not support yet diversions (now #138) and I had some leftover .distrib files for some reason.  if only it was a tiny bit faster ;)